src/Microsoft.PowerShell.Commands.Utility/commands/utility/ConvertFromSddlStringCommand.RightType.Generate.ps1

```diff
@@ -0,0 +1,119 @@
+$RightTypes = @(
```

Comment:
Was this file meant to be checked in?  It's not part of the build to run and the resulting source doesn't indicate it shouldn't be edited.